src/screens/home/HomeScreen.tsx

```diff
@@ -123,8 +124,8 @@ const styles = StyleSheet.create({
     backgroundColor: colors.mainColor.dark,
   },
   text: {
-    fontSize: 20,
-    fontWeight: 'bold',
+    fontFamily: fontTheme.fonts.bold.fontFamily,
+    fontSize: fontTheme.fontSizes.header1.size,
```

Comment:
이렇게 하면 좋을 것 같은데 어떤가요?
```suggestion
    ...fontTheme.header1, // fontFamily, fontWeight, size, lineHeight 포함
```

Reply:
위와 같은 내용의 수정사항 반영이 될 것 같은데, 확인했습니다!